Assets/MixedRealityToolkit.SDK/Inspectors/Input/Handlers/PointerClickHandlerInspector.cs

```diff
@@ -26,13 +27,35 @@ public override void OnInspectorGUI()
         {
             base.OnInspectorGUI();
 
-            if (!MixedRealityInspectorUtility.CheckMixedRealityConfigured(false)) { return; }
+            bool enabled = CheckMixedRealityToolkit();
+            if (enabled)
+            {
+                if (!MixedRealityToolkit.Instance.ActiveProfile.IsInputSystemEnabled)
```

Comment:
The new pattern for accessing profiles and systems has changed. Please see #4315 and #4437 for reference and feel free to reach out with questions.

Reply:
@davidkline-ms , hmm so what service then contains IsInputSystemEnabled as your commet highlighted(aka what service consumes MixedRealityToolkitConfigurationProfile in it's constructor)? The only method I see for accessing profiles is roughly:

```
IMixedRealityService service;
MixedRealityServiceRegistry.TryGetService<IMixedRealityService>(out service); 

svc.profile.blah.dosomething etc
```

Reply:
at this point, there is nothing that consumes that profile, other than the MixedRealityToolkit object.

based on feedback from some customers, there will be scenarios where there isnt a MixedRealityToolkit object in the scene.

many (though not yet all) instances of Is<Service>Enabled are replaceable with a check to see if the system is non-null